Sprint2 Cooked Document: But Again This Time

ChristopherOaks opened the pull request:
Made it not a pdf because the #haters got to me

ChristopherOaks committed:
Added Cooked Document to Documents

Added the document that defines our definition of done, MVP, coding standard, etc. to Documents.

ChristopherOaks committed:
Create CookedDocument

made CookedDocument but again

minknd reviewed (dismissed):
Don't let the haters get you down man

ChristopherOaks committed:
Delete Documentation/COOKED Document.pdf

actually got rid of the pdf this time. #hatersaintgonnagettome #tyler ismeanfornoreason

BrofessorTec reviewed (approved):
igtwwbisah
